src/replicache.test.ts

```diff
@@ -168,7 +168,7 @@ testWithBothStores('put, get, has, del inside tx', async () => {
       await tx.put(key, value);
       expect(await tx.has(key)).to.equal(true);
       const v = await tx.get(key);
-      expect(v).to.eql(value);
+      expect(v).to.deep.equal(value);
```

Comment:
I finally got too tired trying to remember when to use `eq` vs `eql` and read the documentation!